### ISSUE TITLE
Failing to generate output file

### ISSUE THREAD

**Issue by RBin789** [crunchy-labs/crunchy-cli#337]
**Describe the bug**
When I try to download the season everything works fine it downloads both videos with there respective audio files and the subtitle file then as soon as it starts "Generating output file" it crashs with :
```
thread 'tokio-runtime-worker' panicked at D:\a\crunchy-cli\crunchy-cli\crunchy-cli-core\src\utils\download.rs:1199:22:
called `Option::unwrap()` on a `None` value
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
:: An error occurred: task 1252 panicked
```

**To Reproduce**
Just run the command below
```
archive -a en-US -a zh-CN -s en-US --ffmpeg-preset h264 --speed-limit 2MB -o "{series_name} - S{season_number}E{episode_number} - {title}.mkv" https://www.crunchyroll.com/series/GZJH3DJ8E/the-daily-life-of-the-immortal-king[S2]
```

**Expected behavior**
It should download every episode of  The Daily Life of the Immortal King Season 2 with English and Mandarin audio and English Subtitles. with files in the form "The Daily Life of the Immortal King - S02E# - Episode Name"

**Client:**
 - OS: Windows
 - Version crunchy-cli-v3.2.5
 
**Additional context**
Downloading has worked for while but yesterday while downloading (The Daily Life of the Immortal King) my computer shutdown unexpectedly, I think something tried to use Hyper-V-Hypervisor when virtualization was disabled so my computer shutdown anyway after that i went into my BIOS and re enabled virtualization.  Now it gives the error i described above when i try to download it again.  Is crunchy-cli  sensitive to virtualization i don't want to disable it just to use crunchy-cli.

**Comment by bytedream** [comment]
Hey thanks for your issue report. crunchy-cli does nothing with virtualization so this shouldn't be a blocker.
The error occurs because crunchy-cli asserts that ffmpeg writes its progress in a specific format to a file but in your case there seems to be some sort of issue with the progress output ffmpeg reports.

A fix for such issues will land in the next release

**Comment by RBin789** [comment]
Thanks for the response in the meantime is there anything i can do to stop the bug?

**Comment by bytedream** [comment]
You can use the pre-built windows binary from [here](https://github.com/crunchy-labs/crunchy-cli/actions/runs/8022256274) (scroll down and download the `crunchy-cli-windows-x86_64` file). It contains the fix

**Comment by bytedream** [comment]
The fix is now available in [v3.3.0](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.3.0)
